mcs/class/System.Security/System.Security.Cryptography.Xml/corefx/SR.cs

```diff
@@ -0,0 +1,96 @@
+//
```

Comment:
I put it into `System.Security.Cryptography.Xml`, because `System.Security` in Mono contains multiple namespaces, like `.Cryptography.Xml`, `.Cryptography.Pkcs`, etc. In .NET Core, each namespace has its own strings. Although `resx2sr.exe` is able to merge multiple `.resx` files into a single `SR.cs`, I noticed that if there are strings with the same identifier, the value of the first specified `.resx` wins, which means that potentially we may lose important data. Since we cannot guarantee that .NET Core developers will never use strings with same identifiers, I decided to put each `SR.cs` into its own namespace. Are there any problems in this approach?